### DISCUSSION TITLE
Prisma not validate when update

### DISCUSSION THREAD

**Discussion by KozielGPC** [prisma/prisma#10600] [Q&A]
Hello everyone, 
I'm having some problems when using Update method, 
For example, field "name" must be greater than 3 characters, It does validate when creating, but when I update does not and save with wrong type of validated data.
I guess It's something relationed with generators in schema.prisma, but dont found anything yet.

Create Input in /prisma/@generated:

![image](https://user-images.githubusercontent.com/37910437/144899469-3ddf6027-6304-41d4-97ed-3bf9fc51efb4.png)

Update input in /prisma/@generated:

![image](https://user-images.githubusercontent.com/37910437/144899546-652bfb67-c99b-4603-b63b-05d1c8250f31.png)

**Comment by austincrim** [comment]
Hey @KozielGPC 👋,

Is the `@Validator` decorator coming from `@nest/graphql` or somewhere else? What are you using to generate the validation?
